static/js/pages/instances/InstanceTerminal.tsx

```diff
@@ -62,7 +62,7 @@ const InstanceTerminal: FC<Props> = ({ controlTarget, notify }) => {
     }
 
     setTerminalLoading(true);
-    const result = await fetchInstanceExec(name, project, payload).catch(
+    const result = await connectInstanceExec(name, project, payload).catch(
       (e) => {
         setTerminalLoading(false);
```

Comment:
While comparing side-by-side the `InstanceTerminal` and the `InstanceTextConsole`, I just happened to notice that the name of this state `[isTerminalLoading, setTerminalLoading]` sounds a bit "too customised", considering that there is no `isLoading` state in this component and we use `isLoading` for the state of the `TextConsole`.
I suggest renaming it to `[isLoading, setLoading]` for consistency, but it's a very tiny and unimportant thing, so I'll approve and feel free to keep it as it is.